optuna/samplers/tpe/sampler.py

```diff
@@ -371,8 +371,6 @@ def _gmm_log_pdf(
             raise ValueError(
                 "The 'sigmas' should be 2-dimension. " "But sigmas.shape = {}".format(sigmas.shape)
             )
-        _samples = samples
```

Comment:
Asserting `samples.ndim == 1` might aid readability but it's absolutely not a strong opinion (and applies to the above checks raising errors as well).

Reply:
Thank you for reviewing.
On that point, I think the other assertions also need changes (`xyz.ndim != 1` &rarr; `xyz.ndim==2`).
I'll create another PR for the refactoring.